stablehlo/integrations/python/tests/stablehlo.py

```diff
@@ -210,8 +211,8 @@ def test_api_version():
 @run
 def test_serialization_apis():
   curr_version = stablehlo.get_current_version()
-  assert curr_version == "0.10.0"
-  
+  assert bool(re.match('\d+\.\d+\.\d+', curr_version))
```

Comment:
How about adding `^` and `$` to the beginning and the end of the regex correspondingly?